src/store/reducerLogic.js

```diff
@@ -0,0 +1,10 @@
+export const restartGame = {
+  cells: Array(9).fill(null),
+  move: 'X',
+  info: 'Ход: X',
```

Comment:
You should not store presentational logic in redux
`info` contains just a label which can be calculated from a `move`